core/controllers/creator_dashboard.py

```diff
@@ -233,15 +233,25 @@ def _round_average_ratings(rating):
 
             subscribers_list.append(subscriber_summary)
 
+        user_settings = user_services.get_user_settings(
+            self.user_id, strict=False)
+        display_preference = user_settings.creator_dashboard_display
```

Comment:
Hm, could we call the attribute creator_dashboard_display_preference (or creator_dashboard_display_pref)? I know it's long, but the current name doesn't really make it clear that it's a user setting.

Reply:
Done